resources/docker-build_p2p/ssh.rc

```diff
@@ -4,8 +4,11 @@
 ### the IP or DN of the p2p server
 p2p_server=86.119.39.89
 
+### the port of the http service in the p2p server
+http_port=800
```

Comment:
I changed it a little, maybe it makes more sense. IP & HTTP-port are used for initial connection, the Server the says what the ssh port is. Maybe its too confusing to have both in the same File, but this was the easiest way to do it.